controllers/opentelemetrycollector_controller_test.go

```diff
@@ -38,8 +38,9 @@ import (
 	"github.com/open-telemetry/opentelemetry-operator/controllers"
 	"github.com/open-telemetry/opentelemetry-operator/internal/config"
 	"github.com/open-telemetry/opentelemetry-operator/pkg/autodetect"
+	openshiftroutes "github.com/open-telemetry/opentelemetry-operator/pkg/autodetect/openshiftroutes"
```

Comment:
I didn't mean to create a subpackage in the `autodetect` but directly put the file (e.g. `openshiftroutes.go`) to the `autodetect` package.

Reply:
The problem with that is that variables like `Available` and `NotAvailable` will need a prefix like `OpenShiftRoutes` creating values like `OpenShiftRoutesAvailable` and the same for the `Platform` values.

I think this is more idiomatic and easier to manage.

Reply:
y, shouldnt be to hard to rename.

Reply:
Is not about hard/easy to rename. Is about having long names for variables when this can be easier to manage with the packages. But, if you both agree, I'll change it.